From 9ce6403cf68f6fa5c186758a58ba4135bcbae4bd Mon Sep 17 00:00:00 2001
From: Li Jun <b47624@freescale.com>
Date: Wed, 25 Jun 2014 15:28:45 +0800
Subject: [PATCH 0074/5242] MLK-11340-34 Documentation: usb: chipidea: Update
 test procedure for HNP polling support

commit  a3ae4ddc750173580bc4920c016d7d9f271ea29c from
https://source.codeaurora.org/external/imx/linux-imx.git

Update HNP test procedure as HNP polling is supported.

Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 4aac546c4338246b1e55647796badedf6e80bfbe)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/usb/chipidea.txt |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Documentation/usb/chipidea.txt b/Documentation/usb/chipidea.txt
index d1eedc0..7e385dc 100644
--- a/Documentation/usb/chipidea.txt
+++ b/Documentation/usb/chipidea.txt
@@ -32,7 +32,10 @@ cat /sys/kernel/debug/ci_hdrc.0/registers
    B-device should take host role and enumerate A-device.
 
 4) A-device switch back to host.
-   On B-device:
+   On A-device:
+   echo 1 > /sys/bus/platform/devices/ci_hdrc.0/inputs/a_bus_req
+
+   or, on B-device:
    echo 0 > /sys/bus/platform/devices/ci_hdrc.0/inputs/b_bus_req
 
    or, by introducing HNP polling, B-Host can know when A-peripheral wish
-- 
1.7.9.5

